src/plugins/cordova-plugin-device-motion/sim-host.js

```diff
@@ -197,47 +208,58 @@ function createCanvas() {
     ThreeDee.setLight(-300, -300, 800);
 
     node.addEventListener('mousemove', function (e) {
-        if (_mouseDown && !_shiftKeyDown) {
-            _offsets.x = (_offsets.x + _oldX - e.offsetX) % 360;
-            _offsets.y = (_offsets.y + _oldY - e.offsetY) % 360;
-
-            _alpha = _alpha || 0;
-
-            // enforce gamma in [-90,90] as per w3c spec
-            _gamma = -_offsets.x;
-            if (_gamma < -90) {
-                _gamma = -90;
+        if (_mouseDown) {
```

Comment:
That's a great idea @dlebu! yes, the mobile spec app has tests for all of the core plugins.
You should clone the [mobile spec app repo](https://github.com/apache/cordova-mobile-spec) in some location in which you can clone the other repos needed too. This is the main [documentation](https://github.com/apache/cordova-mobile-spec/blob/master/createmobilespec/README.md) for creating the mobile spec app.